apollo-router/src/plugins/telemetry/config.rs

```diff
@@ -56,12 +56,26 @@ pub(crate) struct Conf {
     /// Logging configuration
     #[serde(rename = "experimental_logging", default)]
     pub(crate) logging: Logging,
+
+    #[cfg(feature = "telemetry_next")]
+    #[serde(rename = "logging", default)]
+    pub(crate) new_logging: config_new::logging::Logging,
     /// Metrics configuration
     pub(crate) metrics: Metrics,
     /// Tracing configuration
     pub(crate) tracing: Tracing,
     /// Apollo reporting configuration
     pub(crate) apollo: apollo::Config,
+
+    #[cfg(feature = "telemetry_next")]
+    /// Event configuration
+    pub(crate) events: config_new::events::Events,
```

Comment:
Is an event a log ?

Reply:
Yes, although they only happen as part of a trace.

https://opentelemetry.io/docs/specs/otel/logs/event-api/

The way to think about it is that logs are the output, and events are the things that make it into traces and logs.